src/internal.c

```diff
@@ -3893,10 +3893,11 @@ void FreeX509Name(WOLFSSL_X509_NAME* name)
         {
             int i;
             for (i = 0; i < MAX_NAME_ENTRIES; i++) {
-                if (name->entry[i].set) {
+                if (name->entry[i].object != NULL)
                     wolfSSL_ASN1_OBJECT_free(name->entry[i].object);
+                if (name->entry[i].value != NULL)
                     wolfSSL_ASN1_STRING_free(name->entry[i].value);
-                }
+                XMEMSET(name->entry[i], 0, sizeof(WOLFSSL_X509_NAME_ENTRY));
```

Comment:
Looks like you'll need an `&` here:

```
src/internal.c: In function ‘FreeX509Name’:
./wolfssl/wolfcrypt/types.h:604:46: error: incompatible type for argument 1 of ‘memset’
             #define XMEMSET(b,c,l)    memset((b),(c),(l))
                                              ^
src/internal.c:3900:17: note: in expansion of macro ‘XMEMSET’
                 XMEMSET(name->entry[i], 0, sizeof(WOLFSSL_X509_NAME_ENTRY));
                 ^
```

Reply:
Funny. I had this locally but forgot to put it in the PR commit. Pushed now.